src/main/resources/languages/messages_en.yml

```diff
@@ -155,6 +155,12 @@ messages:
   - '&742. The answer is 42.'
   - '&7A Walshy a day will keep the troubles away.'
   - '&7Never dig straight down!'
+  - '&7Tis but a flesh wound!'
+  - '&7I am &4NOT &7the Messiah!'
+  - '&7Always look on the bright side of life!'
+  - '&7African or European Swallow?'
```

Comment:
I would probably not include this one.

Reply:
you marked all 4 which one out of the 4 you mean?

Reply:
I only marked one, the last one. All above is only automatically added context by GitHub.

Reply:
I figured 5 seconds after